### PR TITLE
LOCUS-49 - add jaxb-xjc/jaxb-impl support added

### PR DESCRIPTION
**Pull request by pleacu**
Add support for jaxb-xjc and jaxb-impl for Fuse Tooling (transformation)

@nickboldt - could you please review?  many thanks!

### CODE REVIEW

File: plugins/org.jboss.tools.locus.jaxb-impl/2.2.7/plugin.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pluginName = JAXB XJC Plug-in
```
Comment (nickboldt) [inline plugins/org.jboss.tools.locus.jaxb-impl/2.2.7/plugin.properties]: This plugin has the same pluginName as plugins/org.jboss.tools.locus.jaxb-xjc/2.2.7/plugin.properties, "JAXB XJC Plug-in". Should this one be different?

File: plugins/org.jboss.tools.locus.jaxb-xjc/2.2.7/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,598 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.jboss.tools.locus.jaxb-xjc;singleton:=true
+Bundle-Version: 2.2.7.qualifier
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Export-Package: com.sun.codemodel.ClassType,
```
Comment (nickboldt) [inline plugins/org.jboss.tools.locus.jaxb-xjc/2.2.7/META-INF/MANIFEST.MF]: Are you exporting PACKAGES, or CLASSES?

File: plugins/org.jboss.tools.locus.jaxb-impl/2.2.7/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+Export-Package: AccessorFactory,
+ AccessorFactoryImpl,
+ AnyTypeAdapter,
+ CycleRecoverable,
+ DatatypeConverterImpl,
+ IDResolver,
+ InternalAccessorFactory,
+ ValidationEventLocatorEx,
+ XmlAccessorFactory,
+ api/AccessorException,
```
Comment (nickboldt) [inline plugins/org.jboss.tools.locus.jaxb-impl/2.2.7/META-INF/MANIFEST.MF]: Should these be period-delimited, not dash-delimited?

File: plugins/org.jboss.tools.locus.jaxb-impl/2.2.7/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -0,0 +1,350 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.jboss.tools.locus.jaxb-impl;singleton:=true
+Bundle-Version: 2.2.7.qualifier
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Export-Package: AccessorFactory,
```
Comment (nickboldt) [inline plugins/org.jboss.tools.locus.jaxb-impl/2.2.7/META-INF/MANIFEST.MF]: Looks like you're exporting CLASSES, not PACKAGES.

**Comment by pleacu** [comment]
@nickboldt  - at your convenience could you take a look at this again??  Thkx!

**Comment by nickboldt** [comment]
merged.

**Pull request closed without merging**